backend/src/main/java/gov/cdc/usds/simplereport/service/model/crm/AccountRequestDynamicsData.java

```diff
@@ -30,11 +30,8 @@ public AccountRequestDynamicsData(final AccountRequest accountRequest) {
     // variableMap.put("bah_state_text", accountRequest.getState()); // need mapping to dynamics
     variableMap.put("bah_zipcode_text", accountRequest.getZip());
     variableMap.put("bah_county_text", accountRequest.getCounty());
-    variableMap.put(
-        "bah_testingsitetype_code",
-        DynamicsValueMapping.convertToCode(Prefix.TST, accountRequest.getFacilityType()));
-    variableMap.put("bah_othertestingsite_text", accountRequest.getFacilityTypeOther());
     variableMap.put("bah_organizationname_text", accountRequest.getOrganizationName());
+    variableMap.put("bah_organizationtype_text", accountRequest.getOrganizationType());
```

Comment:
**praise:** Thanks for figuring out the Dynamics bit!

Reply:
Do we have any control over the Dynamics fields at this point? Setting the main select to "Other" always and then specifying the org type in the "Other Testing Site Type" field feels a bit like a weird workaround.